src/views/CustomToken.vue

```diff
@@ -214,8 +218,9 @@ export default {
         this.isSymbolEditable = Boolean(this.networkAssets.find(_symbol => _symbol === this.symbol))
       }
     }, 500),
-    async selectChain (chain) {
+    async selectChain (chain, nativeAsset) {
       this.chain = chain
+      this.nativeAsset = nativeAsset
```

Comment:
Hi, I think that we don't need to set the nativeAsset manually, we can get it from cryptoassets:
```
import { chains } from '@liquality/cryptoassets'
...
async selectChain (chain) {
 this.chain = chain
 this.nativeAsset = nativeAsset
 const { nativeAsset } = chains[nativeAsset]
 this.nativeAsset = nativeAsset
...
}
```